.travis.yml

```diff
@@ -20,455 +20,346 @@
-                      - libpsl-dev
-                      - libbrotli-dev
-        - os: linux
-          compiler: clang
-          dist: bionic
-          env:
-              - T=tidy
-              - OVERRIDE_CC="CC=clang-7" OVERRIDE_CXX="CXX=clang++-7"
-          addons:
-              apt:
-                  sources:
-                      - *common_sources
-                      - llvm-toolchain-bionic-7
-                  packages:
-                      - *common_packages
-                      - clang-7
-                      - clang-tidy-7
-                      - libpsl-dev
-                      - libbrotli-dev
-        - os: linux
-          compiler: clang
-          dist: bionic
-          env:
-              - T=scan-build
-              - OVERRIDE_CC="CC=clang-7" OVERRIDE_CXX="CXX=clang++-7"
-          addons:
-              apt:
-                  sources:
-                      - *common_sources
-                      - llvm-toolchain-bionic-7
-                  packages:
-                      - *common_packages
-                      - clang-7
-                      - libpsl-dev
-                      - libbrotli-dev
-        - os: linux
-          compiler: clang
-          dist: xenial
-          env:
-              - T=debug CFLAGS="-fsanitize=address,undefined,signed-integer-overflow -fno-sanitize-recover=undefined,integer -Wformat -Werror=format-security -Werror=array-bounds -g" LDFLAGS="-fsanitize=address,undefined -fno-sanitize-recover=undefined,integer" LIBS="-ldl -lubsan"
-              - OVERRIDE_CC="CC=clang-7" OVERRIDE_CXX="CXX=clang++-7"
-          addons:
-              apt:
-                  sources:
-                      - *common_sources
-                      - llvm-toolchain-xenial-7
-                  packages:
-                      - *common_packages
-                      - clang-7
-                      - libpsl-dev
-                      - libbrotli-dev
-        - os: linux
-          arch: arm64
-          compiler: gcc
-          dist: bionic
-          env:
-              - T=debug C="--enable-alt-svc"
-              - OVERRIDE_CC="CC=gcc-8" OVERRIDE_CXX="CXX=g++-8"
-          addons:
-              apt:
-                  sources:
-                      - *common_sources
-                  packages:
-                      - *common_packages
-                      - libpsl-dev
-                      - libbrotli-dev
-                      - libev-dev
-                      - libssl-dev
-                      - libtool
-                      - pkg-config
-                      - zlib1g-dev
+jobs:
+  include:
+  - env:
+    - T=normal C="--with-gssapi --with-libssh2" CHECKSRC=1
+    - OVERRIDE_CC="CC=gcc-8" OVERRIDE_CXX="CXX=g++-8"
+    addons:
+      apt:
+        <<: *common_apt
+        packages:
+        - *common_packages
+        - krb5-user
+        - libssh2-1-dev
+  - env:
+    - T=normal C=--with-libssh
+    - OVERRIDE_CC="CC=gcc-8" OVERRIDE_CXX="CXX=g++-8"
```

Comment:
Focal has GCC 8 as default. Maybe this one shouldn't be manually downgraded to 8.